runtests.jl.in

```diff
@@ -0,0 +1,7 @@
+using Test
+
+include(joinpath("@@GAPJULIA_DIR@@","LibGAP.jl","src","initialization.jl"))
```

Comment:
Why @@ and not just @ as is the usual convention?

Reply:
Basically because Julia has macros that start with `@`, so I wanted to avoid confusion here.